velox/substrait/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ get_filename_component(PROTO_DIR ${substrait_proto_directory}/, DIRECTORY)
 # Generate Substrait hearders
 add_custom_command(
   OUTPUT ${PROTO_OUTPUT_FILES}
-  COMMAND protoc --proto_path ${CMAKE_SOURCE_DIR}/ --cpp_out ${CMAKE_SOURCE_DIR}
+  COMMAND protoc --proto_path ${proto_directory}/ --cpp_out ${PROTO_OUTPUT_DIR}
```

Comment:
Why is this change? Can it be responsible for macos build failure?

Reply:
Krishna Pai change it at commit:52d1dcc14e2a69da2e244b86bfe00edab078aac6  on July 14

Reply:
Actually, looks like this PR reverts Krishna's fix. Remove this change and retry.

Reply:
use   COMMAND protoc --proto_path ${CMAKE_SOURCE_DIR}/ --cpp_out ${CMAKE_SOURCE_DIR} , need  import "velox/substrait/proto/substrait/xxx.proto";

use  COMMAND protoc --proto_path ${proto_directory}/ --cpp_out ${PROTO_OUTPUT_DIR}， need import "substrait/xxx.proto";
But the source code of substrait-io is import "substrait/xxx.proto";

Reply:
I had remove this change and retry.

Reply:
@PenghuiJiao Is this working now ? fyi protoc --proto_path ${CMAKE_SOURCE_DIR}/ --cpp_out ${CMAKE_SOURCE_DIR} should work for all.

Reply:
nit: if you dont quote github turns the `${CMAKE_SOURCE_DIR}` to a wierd font.

Reply:
> @PenghuiJiao
> 
> I'm seeing test failures: Substrait2VeloxPlanConversionTest.q6 and FunctionTest.constructFunctionMap. Are these passing for you?

These tests failed